tools/wake-format/README.md

```diff
@@ -0,0 +1,481 @@
+# wake-format
+
+`wake-format` is a tool for automatically applying the wake style guide to wake source files. It is intentionally opinionated to create a singular correct formatting standard.
+
+# Usage
```

Comment:
```suggestion
## Usage
```

should there be some heading levels?

Reply:
Yeah, I need to figure out the correct nesting. I hit `#####` at one point which I couldn't tell apart from normal text so I un-nested everything by one, I don't think that was the correct choice though. Will work on it! Thanks :)